geode-junit/src/main/java/org/apache/geode/util/test/TestUtil.java

```diff
@@ -63,7 +63,7 @@ private static String getResourcePath(String name, URL resource) {
         File tmpFile = File.createTempFile(filename, null);
         tmpFile.deleteOnExit();
         FileUtils.copyURLToFile(resource, tmpFile);
-        return compatibleWithWindows(tmpFile.getAbsolutePath());
+        return tmpFile.getAbsolutePath();
```

Comment:
Just FYI, I found that `Paths.get(URI)` will return the correct path on both Windows and Unix (but throws an exception if the resource is `jar:file:/...` instead of `file:/...`). But in this piece of code it feels like there would need to be a bunch of nested exception blocks which would make it pretty ugly. However since this is just test code the method could also just throw an `Exception`. The upside would be that the `compatibleWithWindows` method would not be necessary any more.